app/org/sagebionetworks/bridge/models/schedules/TaskScheduler.java

```diff
@@ -0,0 +1,26 @@
+package org.sagebionetworks.bridge.models.schedules;
+
+import java.util.List;
+import java.util.Map;
+
+import org.joda.time.DateTime;
+
+public interface TaskScheduler {
+
+    /**
+     * Get all the tasks for the schedule that start before up to the "until" date and time passed to the method. Tasks
```

Comment:
Can you update the comments to clarify this and to clarify what the events map is? What is a triggering event and how do I specify it when I call the API?